docs/docs/getting-started/first-steps-local.md

```diff
@@ -116,6 +120,140 @@ constellation mini down
+2. Now you can create your cluster and it's nodes. `constellation create` uses the options set in `constellation-conf.yaml`. If you want to manually use [Terraform](../reference/terraform.md) instead, follow the corresponding instructions in [Create Workflow](../workflows/create.md).
+
+  ```shell-session
+  constellation create --control-plane-nodes 1 --worker-nodes 1
+  ```
+
+  The Output should look like the following:
+
+  ```shell-session
+  $ constellation create ...
+  Your Constellation cluster was created successfully.
+  ```
+
+  This will create 2 virtual machines, one worker and one control plane. You can change the numbers of nodes as long as your computer has enough resources.
+
+  :::tip
+
+  If you want to look at and interact with the created virtual machines manually, run
+
+  ```shell-session
+  virsh -c qemu+tcp://localhost:16599/system
+  ```
+
+  which gives you a new `virsh` shell. In this shell, you can run
+  ```shell-session
+  list
+  ```
+   to see the created virtual machines, as well as their status. This should look something like this:
+
+  ```txt
+  Id   Name                       State
+  ------------------------------------------
+  1    constell-control-plane-0   running
+  2    constell-worker-0          running
+  ```
+
+  If you want to connect to one of those machines, run
+
+  ```shell-session
+  console <ID>
+  ```
+
+  where `<ID>` would be the id of your virtual machine as seen above (e.g. `1`). After that, press Enter another time and you should drop into a shell on the virtual machine.
+
+  :::
+
+  :::caution
+
+  Please note that connections via `qemu+tcp` aren't encrypted and shouldn't be used in a security relevant environment!
+
+  :::
+
+3. Finally, initialize the cluster by running
+
+  ```shell-session
+  constellation init
+  ```
+
+  This should return something like the following output:
+
+  ```txt
+  $ constellation init
+  Your Constellation master secret was successfully written to ./constellation-mastersecret.json
+  Initializing cluster ...
+  Your Constellation cluster was successfully initialized.
+
+  Constellation cluster identifier  g6iMP5wRU1b7mpOz2WEISlIYSfdAhB0oNaOg6XEwKFY=
+  Kubernetes configuration          constellation-admin.conf
+
+  You can now connect to your cluster by executing:
+          export KUBECONFIG="$PWD/constellation-admin.conf"
+  ```
+
+### Connect `kubectl` and deploy a sample application
+
+This process is the same as described in the [MiniConstellation](#connect-kubectl) section.
+
+### Terminate your cluster
+
+To terminate your cluster after you finished, run:
+
+```shell-session
+constellation terminate
+```
+
+This will destroy your cluster and clean your workspace. The VM image as well as `constellation-conf.yaml` will be kept to allow reuse.
+
```

Comment:
Instead of referring to another section here, create one unified section for both deployment options.
We could also go with throwing the "Create your cluster" steps of MiniConstellation and QEMU into tabs, similar to how we did it for the cloud providers for IAM creation: https://docs.edgeless.systems/constellation/getting-started/first-steps#create-a-cluster